test/fail_compilation/diag11769.d

```diff
@@ -2,9 +2,9 @@
 TEST_OUTPUT:
 ---
 fail_compilation/diag11769.d(18): Error: diag11769.foo!string.bar called with argument types (string) matches both:
-fail_compilation/diag11769.d(13):     diag11769.foo!string.bar(wstring _param_0)
+fail_compilation/diag11769.d(13):     diag11769.foo!string.bar(wstring _param_0) 
 and:
-fail_compilation/diag11769.d(14):     diag11769.foo!string.bar(dstring _param_0)
+fail_compilation/diag11769.d(14):     diag11769.foo!string.bar(dstring _param_0) 
```

Comment:
Yes, that extra space just looks wrong and is going to consume peoples' time in the future wondering why it is there. It also interferes with the checkin process that removes trailing spaces from lines.